### PR TITLE
Lineage pull request for: skeleton

### PR DESCRIPTION
**Pull request by cisagovbot**
# Lineage Pull Request #

[Lineage] has created this pull request to incorporate new changes found in an
upstream repository:

Upstream repository: [`https://github.com/cisagov/skeleton-python-library.git`](https://github.com/cisagov/skeleton-python-library.git)
Remote branch: `HEAD`

Check the changes in this pull request to ensure they won't cause issues with
your project.

------------

**Note:** *You are seeing this because one of this repository's maintainers has
configured [Lineage] to open pull requests.*

For more information:

🛠 [Lineage] configurations for this project are stored in `.github/lineage.yml`

📚 [Read more about Lineage][Lineage]

[//]: # (lineage:metadata:{&quot;slayed&quot;:true})
[Lineage]: https://github.com/cisagov/action-lineage/ "Lineage GitHub Action"

### CODE REVIEW

**Review by jsf9k — APPROVED** [review]

**Review by dav3r — APPROVED** [review]

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -240,7 +240,10 @@ jobs:
     strategy:
       matrix:
         # Versions supported by AWS and lambci/lambda images
-        python-version: [3.6, 3.7, 3.8]
+        python-version:
+          - 3.6
+          - 3.7
+          - 3.8
```
Comment (jsf9k) [inline .github/workflows/build.yml:246]: Python 3.9 is now supported in AWS Lambda.
```suggestion
        python-version:
          - 3.6
          - 3.7
          - 3.8
          - 3.9
```

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -13,7 +13,10 @@ jobs:
     strategy:
       matrix:
         # Versions supported by AWS and lambci/lambda images
-        python-version: [3.6, 3.7, 3.8]
+        python-version:
+          - 3.6
+          - 3.7
+          - 3.8
```
Comment (jsf9k) [inline .github/workflows/release.yml:19]:
```suggestion
        python-version:
          - 3.6
          - 3.7
          - 3.8
          - 3.9
```

**Review by jsf9k — CHANGES_REQUESTED** [review]

File: .github/workflows/build.yml (hunk 1 above)
Comment (mcdonnnj) [reply .github/workflows/build.yml:246]: However there is no image for [lambci/lambda](https://hub.docker.com/r/lambci/lambda/tags?page=1&ordering=last_updated&name=build-python3.9). I would like to keep changes like that to their own PRs as well.

Comment (jsf9k) [reply .github/workflows/build.yml:246]: Ah, OK.

File: .github/workflows/release.yml (hunk 2 above)
Comment (jsf9k) [reply .github/workflows/release.yml:19]: @mcdonnnj pointed out that there is no Python 3.9 Docker `lambci/lambda` image available yet.

**Review by jsf9k — APPROVED** [review]

**Review by dav3r — APPROVED** [review]
👍

**Pull request merged**